ast/flag.go

```diff
@@ -32,6 +32,9 @@ func (f *flagSetter) Enter(in Node) (Node, bool) {
 }
 
 func (f *flagSetter) Leave(in Node) (Node, bool) {
+	if x, ok := in.(ParamMarkerExpr); ok {
```

Comment:
*ParamMarkerExpr is change from struct to interface, because the old struct definition depends on `types.Datum`

Go can't type assert on a interface:
```
type T interface { ... }
switch x := in.(type) {
    case T:    // compile error
}
```